test/system/700-play.bats

```diff
@@ -180,6 +180,47 @@ EOF
     run_podman 1 container exists $service_container
 }
 
+@test "podman play -- force-remove service-container" {
+    skip_if_remote "service containers only work locally"
+    # regression test for containers/podman/issues/16964
+
+    # Create the YAMl file
+    yaml_source="$PODMAN_TMPDIR/test.yaml"
+    cat >$yaml_source <<EOF
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    app: test
+  name: test_pod
+spec:
+  restartPolicy: "Never"
+  containers:
+  - command:
+    - true
+    image: $IMAGE
+    name: test
+    resources: {}
+EOF
+
+    run_podman kube play --service-container --log-driver=journald $yaml_source
+
+    # The name of the service container is predictable: the first 12 characters
+    # of the hash of the YAML file followed by the "-service" suffix
+    yaml_sha=$(sha256sum $yaml_source)
+    service_container="${yaml_sha:0:12}-service"
+
+    # Remove the service container.
+    run_podman rm -f $service_container
+
+    # Remove the pod and make sure we get a warning.
```

Comment:
Maybe add a comment like, "prior to #17003 this would error out with container not found in DB"? Just to make it clear that this is the actual part of the test. Not worth a re-push for.